PSIPro.ino

```diff
@@ -2037,8 +2070,9 @@ bool checkDelay()
 void set_global_timeout(unsigned long timeout)
 {
   // use 256 to set as "always on"
-  // 256 is already 4 hours, but multiply this up to 16 hours!
-  if (timeout == 256) timeout *= 4;
+  // 256 = ~4 min
```

Comment:
If I were writing the code comments here I'd say:
// 256ms == ~4 mins.  To make the pattern run longer, square the value
// resulting in ~16 hours